portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Susana's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Welcome to My Portfolio:)!</h1>
+      <p>My name is Susana Perez<p> 
+      <h2>About me</h2>
+      <img src="images/download.png" alt="Front facing picture of a woman" style="height:600px;"/>
```

Comment:
don't hard code the style in html. instead you can set a class or id and then specify height in the css file. this is so that you can swap out the css file and change the entire look of the page.

https://www.htmldog.com/guides/css/intermediate/classid/

Reply:
Will do, thank you!